rootfs/api/models/app.py

```diff
@@ -211,6 +211,17 @@ def create(self, *args, **kwargs):  # noqa
             except KubeException:
                 self._scheduler.ns.create(namespace)
 
+            if settings.KUBERNETES_NAMESPACE_DEFAULT_QUOTA_SPEC != '':
+                quota_name = '{}-quota'.format(namespace)
+                self.log(settings.KUBERNETES_NAMESPACE_DEFAULT_QUOTA_SPEC)
+                quota_spec = json.loads(settings.KUBERNETES_NAMESPACE_DEFAULT_QUOTA_SPEC)
+                self.log('creating Quota {} for namespace {}'.format(quota_name, namespace),
+                         level=logging.DEBUG)
+                try:
+                    self._scheduler.quota.get(namespace, quota_name)
```

Comment:
I accept that having default quota is good and i am for it but adding a feature/enhancement which is asking two different tools(deis and kubectl) to manage it is not something most people like. But this can be a first step and may be create an issue to tackle in the update scenario afterwards.